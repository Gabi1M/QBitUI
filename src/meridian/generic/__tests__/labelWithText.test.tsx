import React from 'react';

import { render } from '@testing-library/react';

import LabelWithText from '../labelWithText';

describe('LabelWithText', () => {
    it('should render as expected', () => {
        const result = render(<LabelWithText label='Something' text='Some text' />);
        expect(result.asFragment()).toMatchSnapshot();
    });
});
