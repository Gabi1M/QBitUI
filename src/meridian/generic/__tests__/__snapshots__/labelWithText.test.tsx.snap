// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`LabelWithText should render as expected 1`] = `
<DocumentFragment>
  .emotion-0 {
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-flex-direction: row;
  -ms-flex-direction: row;
  flex-direction: row;
  -webkit-align-items: center;
  -webkit-box-align: center;
  -ms-flex-align: center;
  align-items: center;
}

.emotion-3 {
  font-family: -apple-system,BlinkMacSystemFont,Segoe UI,Roboto,Helvetica,Arial,sans-serif,Apple Color Emoji,Segoe UI Emoji;
  -webkit-tap-highlight-color: transparent;
  color: inherit;
  font-size: inherit;
  line-height: 1.55;
  -webkit-text-decoration: none;
  text-decoration: none;
}

.emotion-3:focus {
  outline-offset: 2px;
  outline: 2px solid #339af0;
}

.emotion-3:focus:not(:focus-visible) {
  outline: none;
}

.emotion-5 {
  font-family: -apple-system,BlinkMacSystemFont,Segoe UI,Roboto,Helvetica,Arial,sans-serif,Apple Color Emoji,Segoe UI Emoji;
  -webkit-tap-highlight-color: transparent;
  color: #868e96;
  font-size: 14px;
  line-height: 1.55;
  -webkit-text-decoration: none;
  text-decoration: none;
}

.emotion-5:focus {
  outline-offset: 2px;
  outline: 2px solid #339af0;
}

.emotion-5:focus:not(:focus-visible) {
  outline: none;
}

<div
    class="emotion-0"
  >
    <div
      class="emotion-1"
    >
      <div
        class="emotion-2 emotion-3"
      >
        Something
      </div>
      <div
        class="emotion-2 emotion-5"
      >
        Some text
      </div>
    </div>
  </div>
</DocumentFragment>
`;
